thrift/contentatom.thrift

```diff
@@ -0,0 +1,5 @@
+include "atoms/tenfour.thrift"
+
+union ContentAtom {
+  1: tenfour.Content tenfour
```

Comment:
:-1: on tenfour, if you need a differentiator then you can use something like the year that is more descriptive: e.g. quiz-2015 or quiz-format-2015.

Cute code names will get confusing as soon as there are changes in the team